python/changelog.py

```diff
@@ -2,6 +2,8 @@
 import json
 import os
 import sys
+sys.path.insert(0, './taichi')
```

Comment:
I still don't know why we need this instead of directly using https://github.com/taichi-dev/taichi/blob/master/python/taichi/make_changelog.py ..

Reply:
Because if we do `from taichi import changelog`, it would use taichi again?

Reply:
I mean the CI just uses `python python/taichi/make_changelog.py`. This should be OK?

Reply:
Yea it's better that we move make_changelog.py out of python folder :P It doesn't have to be released. ;)

Reply:
Sure, I was just preserving the saving function of this script. In this case, I'll modify the make_changelog.py to allow save.